Buildings/Examples/VAVReheat/Controls/Examples/SupplyAirTemperature.mo

```diff
@@ -0,0 +1,50 @@
+within Buildings.Examples.VAVReheat.Controls.Examples;
+model SupplyAirTemperature
+  "Validation of the supply air temperature control"
+  extends Modelica.Icons.Example;
+  Buildings.Examples.VAVReheat.Controls.SupplyAirTemperature conTSup
+    "Controller"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}})));
+  Buildings.Controls.OBC.CDL.Logical.Sources.Pulse uEna(period=2800)
+    "Enable signal"
+    annotation (Placement(transformation(extent={{-80,-70},{-60,-50}})));
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Constant TSupSet(
+    y(final unit="K", displayUnit="degC"),
+    k=13 + 273.15)
+    "Set point"
```

Comment:
I think we're using setpoint throughout the library.

Reply:
This will be addressed through #2225